config/configmaps/files/config.yaml

```diff
@@ -0,0 +1,11 @@
+images:
+  apiServer: quay.io/modh/odh-ml-pipelines-api-server-container:v1.18.0-8
+  artifact: quay.io/modh/odh-ml-pipelines-artifact-manager-container:v1.18.0-8
+  persistentAgent: quay.io/modh/odh-ml-pipelines-persistenceagent-container:v1.18.0-8
+  scheduledWorkflow: quay.io/modh/odh-ml-pipelines-scheduledworkflow-container:v1.18.0-8
+  viewerCRD: quay.io/modh/odh-ml-pipelines-viewercontroller-container:v1.18.0-8
+  cache: registry.access.redhat.com/ubi8/ubi-minimal
+  moveResultsImage: busybox
```

Comment:
we should at a minimum mirror `busybox` to a quay repo to avoid rate limits

Reply:
this is not scope in this pr, addressed in this PR: https://github.com/opendatahub-io/data-science-pipelines-operator/pull/32/files